SpaceX/HomeScreen/HomeViewController.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  HomeViewController.swift
+//  SpaceX
+//
+//  Created by Julia on 11.07.2022.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController {
+    
+    private var scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        return scrollView
+    }()
+    
+    private var companyDescriptionView = BackroundCardView()
+    
+    private var historyLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Histories"
+        label.font = UIFont(name: "Inter-Regular", size: 15)
+        return label
+    }()
+    
+    private var stackView: UIStackView = {
+        let stack = UIStackView()
+        stack.axis = .vertical
+        stack.spacing = 18
+        return stack
+    }()
+    
+    var viewModel = HomeViewModel()
+    
```

Comment:
private
and Dependency injection
use POP and add this viewModel in init 
`init(viewModel: HomeViewModelProtocol = HomeViewModel())`